frontend/src/js/controllers/challengeCreate.js

```diff
@@ -39,8 +43,13 @@
                     
                 }
                 if (vm.input_file) {
+                    vm.isExistLoader = true;
+                    vm.loaderTitle = '';
+                    vm.loaderContainer = angular.element('.exist-team-card');
+                    vm.startLoader('create challenge');
```

Comment:
Why was this added?

Reply:
@galipremsagar  done. Removed the unnecessary lines as I've already initialized them at the start